### PR TITLE
[Quest] 10674 Trapping the Light Fantastic [3.3.5 fix]

### PR DESCRIPTION
**Pull request by ghost**
**Changes proposed:**

-  Method typo change

**Issues addressed:**

Closes #https://github.com/TrinityCore/TrinityCore/commit/92b7c98c018c1b950a45fb639e70871dca739ebd

**Tests performed:**

.quest add 10674
.go zonexy 60.6 69.0 3522

**Known issues and TODO list:**

- [TrinityCore/TrinityCore/issues/26517]

### CODE REVIEW

File: src/server/scripts/World/npcs_special.cpp [hunk 1]
```diff
@@ -780,15 +780,12 @@ class npc_injured_patient : public CreatureScript
             {                                                   //lower max health
                 case 12923:
                 case 12938:                                     //Injured Soldier
-                    me->SetHealth(me->CountPctFromMaxHealth(75));
                     break;
                 case 12924:
                 case 12936:                                     //Badly injured Soldier
-                    me->SetHealth(me->CountPctFromMaxHealth(50));
```
Comment (acidmanifesto) [inline src/server/scripts/World/npcs_special.cpp:787]: Removing the SetHealth makes the cases useless and does nothing

File: src/server/scripts/Outland/zone_blades_edge_mountains.cpp [hunk 2]
```diff
@@ -946,9 +946,9 @@ class npc_oscillating_frequency_scanner_master_bunny : public CreatureScript
             else
             {
                 // Spell 37392 does not exist in dbc, manually spawning
-                me->SummonCreature(NPC_OSCILLATING_FREQUENCY_SCANNER_TOP_BUNNY, me->GetPositionX(), me->GetPositionY(), me->GetPositionZ() + 0.5f, me->GetOrientation(), TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 50s);
-                me->SummonGameObject(GO_OSCILLATING_FREQUENCY_SCANNER, *me, QuaternionData(), 50s);
-                me->DespawnOrUnsummon(50s);
+                me->SummonCreature(NPC_OSCILLATING_FREQUENCY_SCANNER_TOP_BUNNY, me->GetPositionX(), me->GetPositionY(), me->GetPositionZ() + 0.5f, me->GetOrientation(), TEMPSUMMON_TIMED_OR_DEAD_DESPAWN, 230s);
```
Comment (acidmanifesto) [inline src/server/scripts/Outland/zone_blades_edge_mountains.cpp:949]: Why the increase in time?

File: src/server/scripts/Outland/zone_blades_edge_mountains.cpp [hunk 3]
```diff
@@ -967,7 +967,7 @@ class npc_oscillating_frequency_scanner_master_bunny : public CreatureScript
                 if (Player* player = ObjectAccessor::GetPlayer(*me, playerGuid))
                     DoCast(player, SPELL_OSCILLATION_FIELD);
 
-                timer = 3000;
+                timer = 300;
```
Comment (acidmanifesto) [inline src/server/scripts/Outland/zone_blades_edge_mountains.cpp:970]: Why the decrease in time?

**Comment by Shauren** [comment]
This is why you should not create pull requests off your own 3.3.5 branch but from a pr-dedicated one

**Pull request closed without merging**